ffi/blitbuffer.lua

```diff
@@ -1387,14 +1418,18 @@ return a Color value resembling a given level of blackness/gray
 0 is white, 1.0 is black
 --]]
 function BB.gray(level)
-    return Color8(0xFF - floor(0xFF * level))
+    return Color8(bxor(floor(0xFF * level), 0xFF))
 end
 
 -- some generic color values:
-BB.COLOR_BLACK = Color8(0)
 BB.COLOR_WHITE = Color8(0xFF)
-BB.COLOR_GREY = Color8(0x80)
-BB.COLOR_LIGHT_GREY = Color8(0xD0)
+BB.COLOR_GRAY_E = Color8(0xEE)
+BB.COLOR_LIGHT_GRAY = Color8(0xCC)
+BB.COLOR_GRAY_A = Color8(0xAA)
+BB.COLOR_GRAY_9 = Color8(0x99)
+BB.COLOR_GRAY = Color8(0x88)
+BB.COLOR_GRAY_5 = Color8(0x55)
```

Comment:
I think it might be clearer if we call 'em something like:

* `GRAY_5` → `DIM_GRAY`
* `GRAY` →  `DARK_GRAY`
* `GRAY_9` → `WEB_GRAY`
* `GRAY_A` → `GRAY`
* `LIGHT_GRAY`

I plucked those names from <https://cgit.freedesktop.org/xorg/app/rgb/tree/rgb.txt>. The actual shades are slightly different, but should be Close Enough™.